documentation/src/main/asciidoc/topics/proc_configuring_global_persistent_location.adoc

```diff
@@ -0,0 +1,49 @@
+[id='configuring-global-persistent-location_{context}']
+= Configuring Global Persistent Location
+
+{brandname} preserves global state so that it can restore cluster topology and cached data after restart.
+If you enable global state in your {brandname} configuration, you should also configure a suitable location for {brandname} to persist global state.
+
+Global state is enabled by default with {brandname} Server, which persists cluster state to the `{server_home}/server/data` directory so it can perform graceful shutdowns.
+You do not need to configure a global persistent location for remote caches.
```

Comment:
I am not sure I understand this, are you saying you don't need to configure it for a remotely configured store? Or are you talking about for the client, I wasn't quite sure. Or do you mean you don't need to set it when using the server as the default data directory is probably good enough?

Reply:
I mean that you don't need to set a location for the server because it's already configured.